sys-apps/libc-hdrgen/libc-hdrgen-18.0.0_pre20230825.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+inherit cmake flag-o-matic llvm llvm.org python-any-r1 toolchain-funcs
+
+DESCRIPTION="Generates headers for LLVM libc"
+HOMEPAGE="https://libc.llvm.org"
+
+LICENSE="Apache-2.0-with-LLVM-exceptions"
+SLOT="${LLVM_MAJOR}"
+KEYWORDS=""
+IUSE="test"
+
+DEPEND="
+	sys-devel/llvm:${LLVM_MAJOR}
+"
+
+LLVM_COMPONENTS=( llvm libc cmake )
```

Comment:
Do you really have to build it via LLVM? That sounds like a major bad design.

Reply:
Nope GCC works too, but I thought so at first because some code relies on Clang internals (apparently same for GCC). I'll remove this

Reply:
Oh, do you mean llvm/ as root source directory? Yes, I am pretty sure. Also tells you to do that in docs.